tests/test_specs.py

```diff
@@ -194,19 +182,28 @@ def test_squashed_product() -> None:
 
 
 def test_squashed_multiplied_snake_scan() -> None:
-    inst = Line(z, 1, 2, 2) * Squash(
-        Line(y, 1, 2, 2) * ~Line.bounded(x, 3, 7, 2) * Static.duration(9, 2)
+    inst = ConstantDuration(
+        Line(z, 1, 2, 2) * Squash(Line(y, 1, 2, 2) * ~Line.bounded(x, 3, 7, 2)),
+        9,
+        fly=True,
```

Comment:
The original test should now be translated to:

```suggestion
    inst = Line(z, 1, 2, 2) * Squash(
        Line(y, 1, 2, 2) * ~Line.bounded(x, 3, 7, 2) * ConstantDuration(Repeat(2), 9, fly=False)
```